bitnami/keydb/templates/master/statefulset.yaml

```diff
@@ -229,7 +229,7 @@ spec:
             - name: health-scripts
               mountPath: /opt/bitnami/scripts/health
               readOnly: true
-            {{- if .Values.auth.usePasswordFiles }}
+            {{- and .Values.auth.enabled .Values.auth.usePasswordFiles }}
```

Comment:
Accidentally removed the `if`, this should fix the CI/CD pipeline, which was failing at `helm-lint` stage.

```suggestion
            {{- if and .Values.auth.enabled .Values.auth.usePasswordFiles }}
```